apps/site/pages/en/learn/typescript/run.md

```diff
@@ -0,0 +1,49 @@
+---
+title: Running TypeScript with a runner
+layout: learn
+authors: AugustinMauroy
+---
+
+# Running TypeScript with a runner
+
+In the previous article, we learned how to run TypeScript code using transpilation. In this article, we will learn how to run TypeScript code using a runner.
+
+## Running TypeScript code with `ts-node`
+
+[ts-node](https://typestrong.org/ts-node/) is a TypeScript execution environment for node.js. It allows you to run TypeScript code directly in node.js without the need to compile it first. But it's not typechecking your code. So we recommend to type check your code first with `tsc` and then run it with `ts-node` before shipping it.
+
+To use `ts-node`, you need to install it first:
+
+```bash
+npm i -D ts-node
+```
+
+Then you can run your TypeScript code like this:
+
+```bash
+npx ts-node example.ts
+```
```

Comment:
I don't think we should recommend using `npm install --global`, `ts-node` should be installed at the project level and loaded with e.g `node -r ts-node example.ts`. It might be a good time to talk about `node --run`

Reply:
> It might be a good time to talk about node --run

I haven't had time to fiddle with it yet. But I think it `node --run` should be in the resource https://nodejs.org/en/learn/command-line/run-nodejs-scripts-from-the-command-line . and the typescript resource can point to it.

Reply:
for moment keep it as it and after add learning ressource about `--run`